website/screens/components/select/code/SelectCodePage.tsx

```diff
@@ -4,14 +4,12 @@ import QuickNavContainer from "@/common/QuickNavContainer";
 import DocFooter from "@/common/DocFooter";
 import Code from "@/common/Code";
 import Example from "@/common/example/Example";
-import basic from "./code-examples/basicSelect";
-import controlled from "./code-examples/controlledSelect";
-import uncontrolled from "./code-examples/uncontrolledSelect";
-import multiple from "./code-examples/multipleSelect";
-import searchable from "./code-examples/searchableSelect";
-import groups from "./code-examples/groupedSelect";
-import icons from "./code-examples/iconsSelect";
-import optional from "./code-examples/optionalSelect";
+import basic from "./examples/basic";
```

Comment:
What do you think about calling the file basicUsage.ts?

Reply:
I agree! Done.